scripts/docker/docker-pull-async.ps1

```diff
@@ -12,5 +12,3 @@ function DockerPull {
   $results
 }
 
-DockerPull microsoft/windowsservercore
```

Comment:
OK, very similar setup, but linked clones only with with Pro edition. I paid an upgrade to learn that lesson ;-)